src/pac-file-tester.ts

```diff
@@ -92,7 +89,12 @@ const vmContext = (ip: string) => {
   }
 
   const isPlainHostName = (host: string) => {
-    return (host.search('\\\\.') == -1);
+    return host.indexOf(".") === -1;
```

Comment:
This was not working as expected, added additional test against 
`IS_PLAIN_HOSTNAME_PAC` to reflect working behaviour.

Previously, this always returns TRUE.
```
"localhost".search('\\\\.')
-1
"localhost.com".search('\\\\.')
-1
"www.google.com".search('\\\\.')
-1
"foo".search('\\\\.')
-1
```

Reply:
🤦‍♂️ Good catch. I'm sure these are the functions I copied straight out of Firefox.